### ISSUE TITLE
This is not an issue..its a doubt.........can i use this code for multiclass sentence classification?

### ISSUE THREAD

**Issue by thomasnt13** [giuseppebonaccorso/Reuters-21578-Classification#2]

**Comment by giuseppebonaccorso** [comment]
Yes. You should use a softmax output layer (if only one class should be active) and categorical_cross_entropy loss function.
